community/index.md

```diff
@@ -1,2 +1,64 @@
 # Splinter Community
 
+Welcome!
+
+Splinter is an open source project on GitHub. We encourage you to join the
+Splinter community and contribute to this project in the following ways:
+
+**Chat** at [splinterworld.slack.com](http://splinterworld.slack.com).
+Join us on Slack to ask questions, tell us about a bug, or propose changes and
+new features.
+
+**Report bugs**: We track bugs as GitHub issues at
```

Comment:
```suggestion
**Bi-Weekly Engineering Forums**: We host an [open zoom meeting](https://us02web.zoom.us/j/89560362141) to provide brief updates on Splinter development and to provide a way for the community to connect with each other and with us.   Every other Thursday from 8:00 - 9:00 US-Central time.

**Report bugs**: We track bugs as GitHub issues at
```

Reply:
Great idea!  I'll do that in a separate PR so that this one can be merged asap.